### PR TITLE
[Deployments] Adding protocol fee percentages provider task.

### PR DESCRIPTION
**Pull request by jubeira**

### CODE REVIEW

File: pkg/deployments/tasks/20220725-protocol-fee-percentages-provider/test/task.fork.ts [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  before('setup admin', async () => {
+    const DEFAULT_ADMIN_ROLE = await authorizer.DEFAULT_ADMIN_ROLE();
+    admin = await impersonate(await authorizer.getRoleMember(DEFAULT_ADMIN_ROLE, 0), fp(100));
+  });
+
+  context('without permissions', () => {
+    itRevertsSettingFee(FeeType.Yield, fp(0.0157));
+
+    itRevertsSettingFee(FeeType.Swap, fp(0.0126));
+  });
```
Comment (nventuro) [inline pkg/deployments/tasks/20220725-protocol-fee-percentages-provider/test/task.fork.ts:54]: We don't need to remove this, but conceptually it is a bit overkill.

We know the contract does not allow setting these values without permissions, as it has been thoroughly unit-tested. What we want to do rather is perform a big integration check to make sure that the whole provider -> vault -> collector -> authorizer flow will work correctly and doesn't blow up (and also as a simple reminder of which permissions need to be granted).

File: pkg/deployments/tasks/20220725-protocol-fee-percentages-provider/test/task.fork.ts [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+        .connect(admin)
+        .grantRole(await actionId(protocolFeePercentagesProvider, 'setFeeTypePercentage'), admin.address);
+    });
+
+    itSetsFeeCorrectly(FeeType.Yield, fp(0.1537));
+
+    itRevertsSettingFee(FeeType.Swap, fp(0.0857));
+
+    context('with swapFeePercentage permission', () => {
+      before('grant setSwapFeePercentage permission to fees collector', async () => {
```
Comment (nventuro) [inline pkg/deployments/tasks/20220725-protocol-fee-percentages-provider/test/task.fork.ts:68]:
```suggestion
      before('grant setSwapFeePercentage permission to fees provider', async () => {
```

**Review by nventuro — APPROVED** [review]
Very nice, thanks a lot!

File: pkg/deployments/tasks/20220725-protocol-fee-percentages-provider/test/task.fork.ts (hunk 1 above)
Comment (jubeira) [reply pkg/deployments/tasks/20220725-protocol-fee-percentages-provider/test/task.fork.ts:54]: Thanks; I thought about it too, but I decided to include the tests for completion. I'll take it into account for the next fork test.

**Pull request merged**